apps/web/components/OfficialJournalOfIceland/OJOIUtils.ts

```diff
@@ -91,3 +91,31 @@ export const formatDate = (date?: string, df = 'dd.MM.yyyy') => {
     throw new Error(`Could not format date: ${date}`)
   }
 }
+
+export const getStringArrayFromQueryString = (
```

Comment:
You could also look into using next-usequerystate to parseServerside query params

parseAsArrayOf might be something to check out

Reply:
Yes, after looking at the docs this is exactly what I could use.